moveit_planners/trajopt/test/trajectory_test.cpp

```diff
@@ -0,0 +1,70 @@
+/*********************************************************************
+ * Software License Agreement
+ *
+ *  Copyright (c) 2019, PickNik Consulting
+ *  All rights reserved.
+ *
+ *********************************************************************/
+
+/* Author: Omid Heidari, Dave Coleman
```

Comment:
how am i an author? you can remove me. the template you used doesn't need attribution